python/3d/auto_generated/qgs3dmapsettings.sip.in

```diff
@@ -48,12 +48,16 @@ Resolves references to other objects (map layers) after the call to :py:func:`~Q
 
     QgsRectangle extent() const;
 %Docstring
-Returns the 3D scene's 2D extent in project's CRS
+Returns the 3D scene's 2D bounding box of the extent in project's CRS.
+If the 3D scene is not rotated, this is similar to :py:func:`~Qgs3DMapSettings.orientedExtent`. Otherwise, this
+is the bounding box of :py:func:`~Qgs3DMapSettings.orientedExtent`.
+
+.. seealso:: :py:func:`orientedExtent`
 
 .. versionadded:: 3.30
 %End
 
-    void setExtent( const QgsRectangle &extent );
+    void setExtent( const QgsRectangle &newExtent );
```

Comment:
This is an API break -- it would break existing use of the function with named arguments in Python.

Also, my preference is to the simpler name anyway

Reply:
Fixed.